sonar-batch-protocol/src/main/protobuf/batch_report.proto

```diff
@@ -251,3 +251,19 @@ message CoverageDetail {
     repeated int32 covered_line = 2 [packed = true];
   }
 }
+
+message FileDependency {
+  optional int32 to_file_ref = 1;
+  optional int32 weight = 2;
+}
+
+message ModuleDependencies {
```

Comment:
It's not possible to do some streaming as we need to read all data in memory to be able to compute the module dependencies.